resources/share/metainfo/com.github.corna.Vivado.appdata.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2021 Nicola Corna <nicola@corna.info> -->
+<component type="desktop">
+  <id>com.github.corna.Vivado</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <name>Xilinx Vivado Design Suite</name>
+  <summary>Xilinx Design Suite for hardware development</summary>
```

Comment:
I think it needs to be clarified it's not official Xilinx app, at best an installer helper. It should also carry a disclaimer like https://flathub.org/apps/details/com.jetbrains.GoLand does in `description`

Reply:
It is already clarified in the description at line 9 ("This package provides a flatpak wrapper over an official installation. As the Xilinx tools are not redistributable, this package does not contain any Xilinx-licensed material, which is instead downloaded on-the-fly through the official Xilinx installer.")

I have also added the disclaimer, as you suggested.